scripts/functions/Function_Sound.js

```diff
@@ -3453,6 +3459,13 @@ function audio_create_play_queue(_format, _sampleRate, _channels)
         var scriptNode = newSound.scriptNode;
         var max_channels = outputBuffer.numberOfChannels;
 
+        if (scriptNode.sourceBuffers.length <= 0) {
+            const voice = audio_sounds.find(voice => voice.soundid === newSound.handle);
+            if (voice !== undefined) {
+                voice.stop();
+            }
+        }

```

Comment:
Should this exit from the function at this point?

Reply:
Yep, good spot. Added a return.